mbuild/examples/test_examples.py

```diff
@@ -15,6 +16,8 @@
 EXAMPLE_NOTEBOOKS = [f for f in glob.glob('mbuild/examples/*/*.ipynb')]
 
 
+@pytest.mark.skipif(sys.platform in ['win32', 'cygwin'], 
```

Comment:
Probably a good idea.  I'm not sure which platform appveyor uses, so I'll take a guess and if the tests are skipped then we should be good to go.